pkg/models/object.go

```diff
@@ -0,0 +1,119 @@
+package models
+
+//-----------------------------------------------------------------------------------------------------
+// NOTE: the object store is in heavy development, and the locations will likely continue to move
+//-----------------------------------------------------------------------------------------------------
+
+import "context"
+
+const (
+	StandardKindDashboard = "dashboard"
+	StandardKindPlaylist  = "playlist"
+	StandardKindFolder    = "folder"
+
+	// StandardKindDataSource: not a real kind yet, but used to define references from dashboards
+	// Types: influx, prometheus, testdata, ...
+	StandardKindDataSource = "ds"
+
+	// StandardKindPanel: currently used in two ways :( in search it defines a panel within a dashboard
+	// This is also used to refer to a panel plugin with type:  heatmap, timeseries, table, ...
+	StandardKindPanel = "panel"
+
+	// StandardKindTransform: used to show that a dashboard depends on a set of transformations
+	// NOTE: this should likey be replaced with kind:system/type:transform/uid:joinByField or something like that
+	StandardKindTransform = "transform"
```

Comment:
this seems like it's trying to express a different class of information than i ever recall discussing - just from the comment, i don't understand how this would fit into either the "raw" or "structured" kind pattern?

Reply:
Yup -- again, this is in the category of currently using it so we need a constant.  

But yes, transform on its own is not a real "kind"

Reply:
Also note... these definitions are just *moved* from the set of things used in search so that we avoid circular dependencies.  The inconsistent usage of "kind" is an existing issue not introduced in this PR (though it does add more comments describing the situation)